### PR TITLE
Update syntaxe doc

### PR DESCRIPTION
**Pull request by Pbast**

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/XWiki/XWikiSyntaxParagraphs.xml [hunk 1]
```diff
@@ -147,5 +147,6 @@ Paragraphs are text elements separated by 2 or more new lines.
 |Simple paragraph|This is a paragraph|This is a paragraph
 |Paragraph on multiple lines|Paragraph on\\multiple lines|Paragraph on\\multiple lines
 |Two paragraphs|Paragraph one\\\\Paragraph two|Paragraph one\\\\Paragraph two
-|Parametrized paragraph|{{{(% style="text-align:center;color:blue" %)}}}\\Centered and blue paragraph|(% style="text-align:center;color:blue" %)\\Centered and blue paragraph</content>
+|Parametrized paragraph|{{{(% style="text-align:center;color:blue" %)}}}\\Centered and blue paragraph|(% style="text-align:center;color:blue" %)\\Centered and blue paragraph
+|span on some portion of the paragraph content|paragraph with {{{(% style="color:red" %)}}}red{{{(%%)}}} span inside|paragraph with (% style="color:red" %)red(%%) span inside</content>
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/XWiki/XWikiSyntaxParagraphs.xml]: Should probably capitalize the first "span on some..." to be consistent with other table rows.

File: xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/XWiki/XWikiSyntaxGroups.xml [hunk 2]
```diff
@@ -126,7 +126,7 @@
 
 === 2.0 Groups ===
 
-Groups can be used to insert another document inline directly into the current document. This allows for example to insert complex elements inside a list item or inside a table cell. Groups are delimited by the following syntactic elements: ##{{{(((...)))}}}##. One Group can contain another Group and there is no limit of imbrication.
+Groups can be used to insert another document or div section inline directly into the current document. This allows for example to insert complex elements or style inside a list item, inside a table cell or a paragraph. Groups are delimited by the following syntactic elements: ##{{{(((...)))}}}##. One Group can contain another Group and there is no limit of imbrication.
```
Comment (vmassol) [inline xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/XWiki/XWikiSyntaxGroups.xml]: This is not fully correct. DIV only means something when you're outputting to HTML. This documentation should be generic. Or you should say that it's for HTML if you really think it's needed (as in "...another document (or a DIV tag when generating HTML)...").

File: xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/XWiki/XWikiSyntaxGroups.xml [hunk 3]
```diff
@@ -157,5 +157,10 @@ Some embedded paragraph.
 ))) | Cell Three
 
 Next paragraph in the top-level document
-)))</content>
+)))
+|((({{{(% class="MyCustomClass" style="color:blue" %)(((blue paragraphs
```
Comment (vmassol) [inline xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/XWiki/XWikiSyntaxGroups.xml]: we use "myClass" in other examples. Might be better to use the same name.

File: xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/XWiki/XWikiSyntaxParagraphs.xml [hunk 4]
```diff
@@ -147,5 +147,6 @@ Paragraphs are text elements separated by 2 or more new lines.
 |Simple paragraph|This is a paragraph|This is a paragraph
 |Paragraph on multiple lines|Paragraph on\\multiple lines|Paragraph on\\multiple lines
 |Two paragraphs|Paragraph one\\\\Paragraph two|Paragraph one\\\\Paragraph two
-|Parametrized paragraph|{{{(% style="text-align:center;color:blue" %)}}}\\Centered and blue paragraph|(% style="text-align:center;color:blue" %)\\Centered and blue paragraph</content>
+|Parametrized paragraph|{{{(% style="text-align:center;color:blue" %)}}}\\Centered and blue paragraph|(% style="text-align:center;color:blue" %)\\Centered and blue paragraph
+|Span on some portion of the paragraph content|paragraph with {{{(% style="color:red" %)}}}red{{{(%%)}}} span inside|paragraph with (% style="color:red" %)red(%%) span inside</content>
```
Comment (vmassol) [inline xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/XWiki/XWikiSyntaxParagraphs.xml]: Again SPAN is not correct here. Span is only for HTML and this is the XWiki Syntax doc, not the HTML syntax documentation...

**Comment by vmassol** [comment]
@Pbast I think we need to fix those mistakes since this needs to be the documentation for the XWiki Syntax and not for the HTML syntax... This is why that in the current doc we don't speak about DIV or SPAN. We talk about parametrized paragraphs or parametrized groups.

**Comment by mflorea** [comment]
I don't think we need to mention anything about DIVs or SPANs. Wiki syntax is meant for content. Once you start thinking how to generate DIVs or SPANs with wiki syntax it means you're not using it for what is was designed.

**Comment by Pbast** [comment]
Ok I fixed it in my new PR (I didn't know how to fix my commit (already merged)
https://github.com/xwiki/xwiki-platform/pull/545

**Pull request merged**